app/script.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+$(function() {
+
+	$("#name").click(function() {
+		$.get("adjective", function(response){
+			console.log(response);
+ 			var adjective = response.word;
+ 				$("#adjective").text(adjective);
+		});
+
+		$.get("verb", function(response){
+			console.log(response);
+ 			var verb = response.word;
+ 			$("#verb").text(verb);
+		});
+
+		$.get("noun", function(response){
+			console.log(response);
```

Comment:
Once you have a branch of code that works, make a new branch from that one, and remove console logs and extra comments and the like. That's what you'll use for production (e.g. pushing to Heroku and turning in for assignments).